rust/cardano-blockchain-types/src/hashes.rs

```diff
@@ -0,0 +1,142 @@
+//! A bunch of specific hashes.
+
+// cspell: words attrss
+
+use std::str::FromStr;
+
+use catalyst_types::hashes::{Blake2b224Hash, Blake2b256Hash, Blake2bHashError};
+use pallas_crypto::hash::Hash;
+
+/// Defines a new type wrapper for the given hash type.
+macro_rules! define_hashes {
```

Comment:
I think this macro make sense to move out to the `catalyst-types` crate

Reply:
I don't mind moving it, but currently it is an implementation detail (private for other code) and it is very specific, so I have doubts about it being reused elsewhere.